README.md

```diff
@@ -22,7 +41,7 @@ HIDAPI has five back-ends:
 On Linux, either the hidraw or the libusb back-end can be used. There are
 tradeoffs, and the functionality supported is slightly different.
 
-Linux/hidraw (linux/hid.c):
+Linux/hidraw (`linux/hid.c`):
```

Comment:
The newline isn't present in the rendered version. And I'd make this subheading **fat**